lib/folderFlows.js

```diff
@@ -38,7 +38,7 @@ function updateFolderFlows (folderId, which, enabled, callback) {
     }
     flows.forEach(flow => {
       count++
-      updateFlowEnabled(flow.id, enabled)
+      updateFlowEnabled(flow, enabled)
```

Comment:
thanks for debugging this.
I would change `getFolderFlows` so it returns flow not ids. As it was and name implies. Do you mind making that change. I am not able to do that atm.
Also please bump app version.

Thank you!